openedx/core/djangoapps/user_api/views.py

```diff
@@ -1,5 +1,10 @@
 """HTTP end-points for the User API. """
 
+import urllib
+import urllib2
+import json
+
+from django.conf import settings
```

Comment:
Please do not forget to remove these imports from here once you move the logic to forms.
Also:
1. The imports are not in order, when you import in the forms please take care of import order
2. Let's only import the functions from these modules that we are going to use in these files, instead of importing the whole module